mousestyles/path_diversity/smooth_noise.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import print_function, absolute_import, division
+
+import pandas as pd
+import numpy as np
+from path_features import compute_angles
+from detect_noise import detect_noise
+
+
+def smooth_noise(movement, paths, angle_threshold, delta_t):
+    r"""
+    Return a new movement pandas DataFrame object containing
+    CT, CX, CY coordinates and the homebase status. The points at which
+    noise, as defined by the input parameters, is detected have been
```

Comment:
Could be confusing explanations. Noise is defined within this function and not specified in the input parameters? Could you specify your words?